src/paperqa/prompts.py

```diff
@@ -113,10 +113,12 @@
     " Respond with the following JSON format:"
     '\n\n{{\n  "summary": "...",\n  "relevance_score": "..."\n  "used_images"\n}}'
     "\n\nwhere `summary` is relevant information from the text - {summary_length} words."
-    " `relevance_score` is an integer 1-10 for the relevance of `summary` to the question."
+    " `relevance_score` is an integer 0-10 for the relevance of `summary` to the question."
```

Comment:
Should we add the `0-10` to the `summary_prompt` above as well?

Reply:
Since we use "not applicable" phrasing here, we don't need 0-10

I documented this in https://github.com/Future-House/paper-qa/pull/1107